crates/typst-library/src/math/lr.rs

```diff
@@ -155,17 +171,27 @@ pub fn abs(
 /// ```
 #[func]
 pub fn norm(
+    /// The size of the brackets, relative to the height of the wrapped content.
+    #[named]
+    #[default(Smart::Auto)]
+    size: Smart<Rel<Length>>,
     /// The expression to take the norm of.
     body: Content,
 ) -> Content {
-    delimited(body, '‖', '‖')
+    delimited(body, '‖', '‖', size)
 }
 
-fn delimited(body: Content, left: char, right: char) -> Content {
+fn delimited(
+    body: Content,
+    left: char,
+    right: char,
+    size: Smart<Rel<Length>>,
+) -> Content {
     LrElem::new(Content::sequence([
         TextElem::packed(left),
         body,
         TextElem::packed(right),
     ]))
+    .with_size(size)
```

Comment:
The `size` arguments on the individual functions should be `Option<Smart<Rel<Length>>>` without `#[default]`. Then, they will be `None` if not present. And here, you would then only `push_size` to the `LrElem` if they are `Some(_)`. By not pushing them at all if an argument wasn't given, `abs(x)` will continue to respect `set math.lr(size: ..)` rules instead of forcing it's default. The same happens in the auto-generated `lr` constructor: The field is only pushed if present and the annotated default has lower precedence than set rules.